src/components/layer-manager/layer-manager.vue

```diff
@@ -22,7 +22,7 @@ const mapStore = useMapStore()
 const appStore = useAppStore()
 const styles = useMvtStyles()
 const sliderStore = useSliderComparatorStore()
-const { bgLayer } = storeToRefs(mapStore)
+const { layers_3d, bgLayer } = storeToRefs(mapStore)
 const { sliderActive } = storeToRefs(sliderStore)
 
 const layers = computed(() => [...mapStore.layers].reverse())
```

Comment:
There should be a computed const for `layers_3d` (list display in the tree is reversed)